lib/core/cardano-wallet-core.cabal

```diff
@@ -195,6 +194,7 @@ test-suite unit
       -threaded -rtsopts
       -Wall
       "-with-rtsopts=-M2G"
+      "-with-rtsopts=-N4"
```

Comment:
You could use`-N` without a number to select the number of available cores on the system which is running the test.
I was surprised to find that hspec does not call `setNumCapabilities` when the `hspec --jobs` arg is used.

Reply:
> I was surprised to find that hspec does not call setNumCapabilities when the hspec --jobs arg is used.

Me too. Hence the RTS option, although I've tried `-N` but it actually made the test slower for me (while also incapacitating my laptop for 5 minutes :sweat_smile: ... Maybe was due to a lack of RAM when trying to run on all CPU threads.